### PR TITLE
Develop Footer with Tailwind CSS

### LINKED ISSUE #50: Migrate from ant design to tailwind css
**Is your feature request related to a problem? Please describe.**
- Since we decided to use tailwind CSS we need to change the existing components from ant design to tailwind css

**Describe the solution you'd like**
- Create the existing components in tailwind css
- Remove ant design from the project

### PR DESCRIPTION
**Pull request by vishwaikon**
## Purpose
Develop the Footer with Tailwind CSS ensuring responsiveness for issue fix #50

## Goals
Move Antd Components to Tailwind CSS

## Approach
N/A

## Screenshots
image

##  Checklist
- [X ] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [ X] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [ X] My code follows the style guidelines of this project
- [ X] I have performed a self-review of my own code
- [X ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Related PRs
N/A

## Test environment
N/A

## Learning
N/A

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -4,6 +4,9 @@ export const theme = {
   extend: {
     colors: {
       'primary-blue': '#1677FF',
+      'darkmod-blue': '#3D317C',            //dark moderate blue
+      'desatdark-blue':'#6D7893',            //desaturated dark blue
+
```
Comment (anjula-sack) [inline tailwind.config.js]:
```suggestion
```

File: src/assets/svg/Icons/Discourseicon.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+const Discourseicon: React.FC = () => {
```
Comment (anjula-sack) [inline src/assets/svg/Icons/Discourseicon.tsx]: Let's rename like DiscourseIcon

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Good job @vishwaikon!  Some minor changes are requested. Also link the issue number to the PR like Fix #<issue-number>

**Comment by anjula-sack** [comment]
@vishwaikon Rename the file names also. If git is not detecting the change check this out: https://stackoverflow.com/questions/13805750/git-fails-to-detect-renaming

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Can you rename the file names as well? DiscourseIcon.tsx also link the issue to the pr description.

**Comment by vishwaikon** [comment]
@anjula-sack  Made the changes and updated the PR description. Regards!

File: src/assets/svg/Icons/FacebookIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const FacebookIcon: React.FC = () => {
+  return (
+    <div>
+      {' '}
```
Comment (anjula-sack) [inline src/assets/svg/Icons/FacebookIcon.tsx]:
```suggestion
```

File: src/assets/svg/Icons/InstagramIcon.tsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        </g>
+        <g
+          id="SVGRepo_tracerCarrier"
+          strokeLinecap="round"
+          strokeLinejoin="round"
+          stroke="#c12584CCCCCC"
+          strokeWidth="0.048"
+        ></g>
+        <g id="SVGRepo_iconCarrier">
+          {' '}
```
Comment (anjula-sack) [inline src/assets/svg/Icons/InstagramIcon.tsx]:
```suggestion
```

File: src/assets/svg/Icons/InstagramIcon.tsx [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+          <path
+            fillRule="evenodd"
+            clipRule="evenodd"
+            d="M12 18C15.3137 18 18 15.3137 18 12C18 8.68629 15.3137 6 12 6C8.68629 6 6 8.68629 6 12C6 15.3137 8.68629 18 12 18ZM12 16C14.2091 16 16 14.2091 16 12C16 9.79086 14.2091 8 12 8C9.79086 8 8 9.79086 8 12C8 14.2091 9.79086 16 12 16Z"
+            fill="#c12584"
+          ></path>{' '}
+          <path
+            d="M18 5C17.4477 5 17 5.44772 17 6C17 6.55228 17.4477 7 18 7C18.5523 7 19 6.55228 19 6C19 5.44772 18.5523 5 18 5Z"
+            fill="#c12584"
+          ></path>{' '}
```
Comment (anjula-sack) [inline src/assets/svg/Icons/InstagramIcon.tsx]:
```suggestion
          ></path>
```

File: src/assets/svg/Icons/YoutubeIcon.tsx [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+          <title>youtube [#ffffff]</title> <desc>Created with Sketch.</desc>{' '}
+          <defs> </defs>{' '}
+          <g
+            id="Page-1"
+            stroke="none"
+            strokeWidth="1"
+            fill="none"
+            fillRule="evenodd"
+          >
+            {' '}
```
Comment (anjula-sack) [inline src/assets/svg/Icons/YoutubeIcon.tsx]:
```suggestion
```

File: src/assets/svg/Icons/YoutubeIcon.tsx [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+        fill="#000000"
+      >
+        <g id="SVGRepo_bgCarrier" strokeWidth="0"></g>
+        <g
+          id="SVGRepo_tracerCarrier"
+          strokeLinecap="round"
+          strokeLinejoin="round"
+        ></g>
+        <g id="SVGRepo_iconCarrier">
+          {' '}
```
Comment (anjula-sack) [inline src/assets/svg/Icons/YoutubeIcon.tsx]:
```suggestion
```

File: src/assets/svg/Icons/YoutubeIcon.tsx [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+          <title>youtube [#ffffff]</title> <desc>Created with Sketch.</desc>{' '}
+          <defs> </defs>{' '}
```
Comment (anjula-sack) [inline src/assets/svg/Icons/YoutubeIcon.tsx]:
```suggestion
          <defs></defs>
```

File: src/assets/svg/Icons/YoutubeIcon.tsx [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+      >
+        <g id="SVGRepo_bgCarrier" strokeWidth="0"></g>
+        <g
+          id="SVGRepo_tracerCarrier"
+          strokeLinecap="round"
+          strokeLinejoin="round"
+        ></g>
+        <g id="SVGRepo_iconCarrier">
+          {' '}
+          <title>youtube [#ffffff]</title> <desc>Created with Sketch.</desc>{' '}
```
Comment (anjula-sack) [inline src/assets/svg/Icons/YoutubeIcon.tsx]:
```suggestion
          <title>youtube [#ffffff]</title> <desc>Created with Sketch.</desc>
```

File: src/assets/svg/Icons/YoutubeIcon.tsx [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+                {' '}
+                <path
+                  d="M251.988432,7291.58588 L251.988432,7285.97425 C253.980638,7286.91168 255.523602,7287.8172 257.348463,7288.79353 C255.843351,7289.62824 253.980638,7290.56468 251.988432,7291.58588 M263.090998,7283.18289 C262.747343,7282.73013 262.161634,7282.37809 261.538073,7282.26141 C259.705243,7281.91336 248.270974,7281.91237 246.439141,7282.26141 C245.939097,7282.35515 245.493839,7282.58153 245.111335,7282.93357 C243.49964,7284.42947 244.004664,7292.45151 244.393145,7293.75096 C244.556505,7294.31342 244.767679,7294.71931 245.033639,7294.98558 C245.376298,7295.33761 245.845463,7295.57995 246.384355,7295.68865 C247.893451,7296.0008 255.668037,7296.17532 261.506198,7295.73552 C262.044094,7295.64178 262.520231,7295.39147 262.895762,7295.02447 C264.385932,7293.53455 264.28433,7285.06174 263.090998,7283.18289"
+                  id="youtube-[#ffffff]"
+                >
+                  {' '}
+                </path>{' '}
+              </g>{' '}
+            </g>{' '}
+          </g>{' '}
```
Comment (anjula-sack) [inline src/assets/svg/Icons/YoutubeIcon.tsx]: Remove spaces {' '} check other places as well

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Review by anjula-sack — APPROVED** [review]
Good job! @vishwaikon

**Pull request merged**